index.js

```diff
@@ -21,15 +21,14 @@ var PN532_COMMAND_INLISTPASSIVETARGET = 0x4A;
 var PN532_COMMAND_GETFIRMWAREVERSION = 0x02;
 var PN532_COMMAND_SAMCONFIGURATION = 0x14;
 var PN532_COMMAND_INDATAEXCHANGE = 0x40;
-var PN532_I2C_READY = 0x01;
 var PN532_PREAMBLE = 0x00;
 var PN532_STARTCODE1 = 0x00;
 var PN532_STARTCODE2 = 0xFF;
 var PN532_POSTAMBLE = 0x00;
 var PN532_I2C_ADDRESS = 0x48 >> 1;
 var PN532_I2C_READBIT = 0x01;
-var PN532_I2C_BUSY = 0x00;
-var PN532_I2C_READY = 0x01;
+var PN532_I2C_READY = 0x00;
+var PN532_I2C_BUSY = 0x01;
```

Comment:
These two variables were annoyingly named because the logic state of each of them was the opposite as was declared. For example, `PN532_I2C_BUSY` is actually true when the IRQ line is pulled high, therefore it should have a value of 1.